### DISCUSSION TITLE
Documentation - especially Create AI-Polygon

### DISCUSSION THREAD

**Discussion by HoLue** [wkentaro/labelme#1489] [Q&A / Help]
I'm looking for a documentation of the software. Fir example I don't know how to use the "Create AI-Polygon". When I select it and I click somewhere a polygon is set and an additional point where I clicked. When I click somewhere else, another od enlarged region is set as polygon and an additional point is set. What does this mean? Can I influence the selection by clicking on different points? How?
Here I try to make my question more clear.
![Step01](https://github.com/user-attachments/assets/bfd7c800-9335-4827-98fe-fc0fc299c643)
![Step02](https://github.com/user-attachments/assets/42fa4644-f549-488b-9f36-839e29ec6586)
![Step03](https://github.com/user-attachments/assets/0041888e-c860-42ba-aeeb-fb34234acc99)
Maybe there are some videos or something like that, too, but I could not find them.

**Comment by maplelost** [comment]
你可以看看这个
https://blog.csdn.net/weixin_44586964/article/details/142591648?spm=1001.2014.3001.5501
